tool-populate/api/Roles.js

```diff
@@ -23,13 +24,12 @@ function loadRoles(options, callback) {
 }
 
 function createRole(roleToCreate) {
-  var roleACL = new Parse.ACL();
-  roleACL.setPublicReadAccess(true);
-
+  var acl = new Parse.ACL()
+  acl.setPublicReadAccess(false)
```

Comment:
True. I suppose I was thinking of an instance where someone would maliciously modify the role object. Couldn't someone, in theory, gain access to the roles and swap the names or make another change? Then all regular users would have admin access.